formula-android-compose/build.gradle

```diff
@@ -1,10 +1,7 @@
 apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
-apply plugin: 'kotlin-kapt'
 apply plugin: "com.github.ben-manes.versions"
```

Comment:
You shouldn't have to apply this plugin to each module, but rather to the top level build.gradle.

Reply:
Likely did this incorrectly, will update in a follow-up